component-samples/demo/aio/java.security

```diff
@@ -0,0 +1,1209 @@
+#
```

Comment:
Considering we are not using soft HSM, this file is not used anymore and can be removed.